package/patterns-yast.spec

```diff
@@ -235,21 +235,23 @@ Provides:       pattern-visible()
 
 Recommends:     yast2-devtools
 Recommends:     yast2-testsuite
+Recommends:     osc
+Recommends:     rubygem(%{rb_default_ruby_abi}:yast-rake)
+Recommends:     rubygem(%{rb_default_ruby_abi}:byebug)
+Recommends:     rubygem(%{rb_default_ruby_abi}:rspec)
+Recommends:     rubygem(%{rb_default_ruby_abi}:simplecov)
+# only in OBS YaST:Head, but install it when available
+# YaST currently requires version 0.41.2
+Recommends:     rubygem(%{rb_default_ruby_abi}:rubocop) = 0.41.2
+Suggests:       yast2-ycp-ui-bindings-devel
+Suggests:       yast2-pkg-bindings-devel-doc
```

Comment:
I think that local doc can be removed completelly

Reply:
It's `Suggests` only, i.e. not installed by default. I think it's OK to have it here, the users will see the package in the list and can decide whether to install or not.

Reply:
well, but we do not have here other doc packages we have. BTW why we do not drop it?

Reply:
Because it's a C++ package, for Ruby packages you can use the online generated doc, e.g. http://www.rubydoc.info/github/yast/yast-yast2, AFAIK there is nothing like that for doxygen.

Oh, maybe we could use the same trick as for https://yast-ui-bindings.surge.sh/....

Reply:
Removed, lets use the same trick here later...